diff -Naur --color android-tools-35.0.2/vendor/extras/libjsonpb/parse/jsonpb.cpp n_android-tools-35.0.2/vendor/extras/libjsonpb/parse/jsonpb.cpp
--- android-tools-35.0.2/vendor/extras/libjsonpb/parse/jsonpb.cpp	2024-08-29 11:46:57.000000000 -0600
+++ n_android-tools-35.0.2/vendor/extras/libjsonpb/parse/jsonpb.cpp	2025-07-14 11:44:37.459863668 -0600
@@ -33,7 +33,7 @@
 static constexpr char kTypeUrlPrefix[] = "type.googleapis.com";
 
 std::string GetTypeUrl(const Message& message) {
-  return std::string(kTypeUrlPrefix) + "/" + message.GetDescriptor()->full_name();
+  return std::string(kTypeUrlPrefix) + "/" + std::string(message.GetDescriptor()->full_name());
 }
 
 ErrorOr<std::string> MessageToJsonString(const Message& message) {
